integration-tests/pages/createReport/userNotVerifiedPage.js

```diff
@@ -0,0 +1,10 @@
+const page = require('../page')
+
+const userNotVerifiedPage = username =>
+  page(`${username} name has not verified their email address`, {
```

Comment:
Is this right? `${username} name has not verified their email address`
Should it be `${username} has not verified their email address` perhaps?